sale_order_product_assortment/models/sale_order.py

```diff
@@ -6,52 +6,31 @@
 class SaleOrder(models.Model):
     _inherit = "sale.order"
 
-    whitelist_product_ids = fields.Many2many(
+    allowed_product_ids = fields.Many2many(
```

Comment:
Would a migration script be needed to rename this column and keep the data?

Reply:
No because this field is computed no store.

Reply:
Oh right, sorry :man_facepalming: